doc/classes/SpinBox.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="SpinBox" inherits="Range" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Numerical input text field.
+		An editor for integers and floating point numbers.
```

Comment:
Again, "editor" is too much of an exaggeration.
I think the old description is fine
```suggestion
		Numerical input field.
```
if anything.